src/main/resources/swagger/api-docs.yaml

```diff
@@ -2037,7 +2177,7 @@ paths:
             {
               "type": "array",
               "items": {
-                "type": "string"
+                $ref: '#/definitions/BillingProjectMembership'
```

Comment:
see the pending rawls change

On Saturday, August 6, 2016, Greg Rushton <notifications@github.com> wrote:

> In src/main/resources/swagger/api-docs.yaml
> <https://github.com/broadinstitute/firecloud-orchestration/pull/239#discussion_r73788886>
> :
>
> > @@ -2037,7 +2177,7 @@ paths:
> >              {
> >                "type": "array",
> >                "items": {
> > -                "type": "string"
> > +                $ref: '#/definitions/BillingProjectMembership'
>
> The change to BillingProjectMembership doesn't seem to be reflected in
> the actual response. For example, the response model schema is shown as:
>
> [
>   {
>     "projectName": "string",
>     "role": "string"
>   }
> ]
>
> However, the actual response is just an array of strings:
> ``
> [
> "broad-dsde-alpha",
> "broad-dsde-dev"
> ]
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/broadinstitute/firecloud-orchestration/pull/239/files/99527ccd3ba585cee29acf25afe3f5d8251fc1c6#r73788886>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABc2tQfvVBjUPLFOoRV3m1emBXeiXsrZks5qdL-MgaJpZM4JeH-s>
> .
>